src/index.js

```diff
@@ -135,6 +135,15 @@ function generate(dataFrom, config = defaultConfig) {
 
   // Grab all required comments into grouped objects
   const notedObjects = collectAllNotedObjects(dataFrom);
+  
+  // Check for unclassified jsdocs
+  if(notedObjects.unclassified.length) {
+	notedObjects.unclassified.forEach((obj) => {
+	  console.error("Found unclassified jsdoc: " + obj.definition);
```

Comment:
I'm going to follow the pattern used by the rest of this file and use console.error.  But I'll also call exit(1) instead of process.exitCode = 1.

Reply:
Actually, I discovered something interesting.  Using exit(1) aborts the whole thing midstream, so I'm getting:
[11:00:04] Starting 'componentUIDefinitions'...
Found unclassified jsdoc: DrawingEdit.prototype.reorderLayer = function (activator, layer)
Found unclassified jsdoc: CIQ.UI.Layout.prototype.showPeriodicity = function (stx, params)
[11:00:04] The following tasks did not complete: packager, chartiq, standard, advanced, components, deprecated, addons, componentUIDefinitions
[11:00:04] Did you forget to signal async completion?
npm ERR! code ELIFECYCLE

whereas setting the exitCode allows everything else to complete.  What is the deisred behavior?  At first we said the former, but I see how annoying it could be that you must fix everything before you can even use load the page.

Reply:
Please check what that does to the packager commands first, the other places in the file are for the CLI and not the exported generate function that is used when streaming. We do not want to exit in the middle.

Reply:
OK, I won't change anything here.